app/assets/javascripts/components/autocomplete.js

```diff
@@ -84,5 +86,52 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     $select.id = $select.id + '-select'
   }
 
+  Autocomplete.prototype.initAutoCompleteWithoutNarrowingResults = function () {
+    // Read options and associated data attributes and feed that as results for inputValueTemplate
+    var $select = this.$module.querySelector('select')
+
+    if (!$select) {
+      return
+    }
+
+    var $options = $select.querySelectorAll('option')
+    var $selectedOption = $select.querySelector('option[selected]')
+    if (!$selectedOption) {
+      $selectedOption = $options[0]
+    }
+    var defaultValue = $selectedOption.textContent
+
+    new window.accessibleAutocomplete({ // eslint-disable-line no-new, new-cap
+      id: $select.id,
+      name: $select.name,
+      element: this.$module,
+      showAllValues: true,
+      defaultValue: defaultValue,
+      autoselect: false,
+      dropdownArrow: function (config) {
+        return '<svg class="' + config.className + '" style="top: 8px;" viewBox="0 0 512 512" ><path d="M256,298.3L256,298.3L256,298.3l174.2-167.2c4.3-4.2,11.4-4.1,15.8,0.2l30.6,29.9c4.4,4.3,4.5,11.3,0.2,15.5L264.1,380.9  c-2.2,2.2-5.2,3.2-8.1,3c-3,0.1-5.9-0.9-8.1-3L35.2,176.7c-4.3-4.2-4.2-11.2,0.2-15.5L66,131.3c4.4-4.3,11.5-4.4,15.8-0.2L256,298.3  z"/></svg>'
+      },
+      source: function (query, syncResults) {
+        var results = []
+        $options.forEach(function ($el) {
+          results.push($el.textContent)
+        })
+        syncResults(results)
+      },
+      onConfirm: function (result) {
+        var value = result && result.value
+        var options = [].filter.call($select.options, function (option) {
```

Comment:
should this be $options?

Reply:
it's an array of items not node elements, that's why I didn't used the $ sign. I share your view on this prefix, for me they're all objects so I will raise it with the other frontend devs and we may get rid of it.

Reply:
Sorry I actually meant whether $select.options should reference the $options variable rather than the $select.options (as per the method above)

Thought the other part is a good point too 😉

Reply:
We're removing the `$select` element from the DOM, but the object is still in the memory and good to be referenced. I updated to reference the `$options` to avoid any doubt.